docs/ybm_cluster_connection-pooling.md

```diff
@@ -0,0 +1,39 @@
+## ybm cluster connection-pooling
+
+Manage Connection Pooling
```

Comment:
docs shouldn't be available for features under FF (disable FF and do a make doc and they'll get deleted)

Reply:
This one still gets generated. The other two don't (the one for enable and disable)

Reply:
it doesn't if you add it only if Ff is enabled in cluster.go. Check PITR config command in that file